examples/async-examples.js

```diff
@@ -0,0 +1,59 @@
+(async () => {	
+	const revai = require('../../revai-node-sdk');
+
+	/* Initialize your client with your revai access token */
+	const token = '<YOUR-ACCESS-TOKEN>';
+	var client = new revai.RevAiApiClient(token);
+
+	/* Get account details */
+	var account = await client.getAccount();
+	console.log(`Account: ${account.email}`);
+	console.log(`Balance: ${account.balance_seconds} seconds`);
+
+	
+	const jobOptions = {
+	    metadata: "This is an example",
+	    skip_diarizationh: true
+	};
+
+	/* Media may be submitted from a local file or from a url by using:
+	 * client.submitJobUrl("https://www.rev.ai/FTC_Sample_1.mp3");
+	 */
+	var job = await client.submitJobLocalFile("./example.mp3", jobOptions);
+
+	console.log(`Job Id: ${job.id}`);
+	console.log(`Status: ${job.status}`);
+	console.log(`Created On: ${job.created_on}`);
+
+	/* Check the status of job */
+	var jobDetails = await client.getJobDetails(job.id);
+	console.log(`Status for job ${job.id} is ${jobDetails.status}`);
+
+	/* Get # of jobs started after a certain job id
+	 * client.getListOfJobs() will default to returning all jobs
+	 */
+	var listOfJobs = await client.getListOfJobs(5, job.id);
+
+	/* Get transcript as plain text
```

Comment:
The code directly below the comment does not get the transcript as plain text. It loops with a 5 second delay until the job is in a terminal state.